src/index.ts

```diff
@@ -1,16 +1,17 @@
+import * as crypto from "crypto";
+import { ECPairFactory } from "ecpair";
+import { bech32m } from "bech32";
+import * as bitcoin from "bitcoinjs-lib";
 import ecc from "./noble_ecc";
-const crypto = require("crypto");
-const ECPairFactory = require("ecpair").ECPairFactory;
-const { bech32m } = require("bech32");
+
 const ECPair = ECPairFactory(ecc);
-const bitcoin = require("bitcoinjs-lib");
 bitcoin.initEccLib(ecc);
 
 type UTXO = {
   txid: string;
   vout: number;
   WIF: string;
```

Comment:
something to think about:  if we change api from requiring WIF to requiring simply privkey, we can drop `ecpair` as a dependency

Reply:
I like it, we can use ecpair it tests and docs, but remove it from dependencies.